py/src/app/pcasts/models/recommendation.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import UniqueConstraint
+from . import *
+
+class Recommendation(Base):
+  __tablename__ = 'recommendations'
+  __bind_key__ = 'db'
+  __table_args__ = (
+      UniqueConstraint('user_id', 'episode_id'),
+  )
+
+  id = db.Column(db.Integer, primary_key=True)
+  user_id = db.Column(db.Integer, db.ForeignKey('users.id', ondelete='CASCADE'))
+  episode_id = db.Column(db.Integer, nullable=False)
+
+  user = db.relationship('User', cascade='all,delete')
```

Comment:
Not sure if the `cascade='all,delete'` is correct here
Haven't tested it with `recommendation` or `bookmark`, but I realized with `subscription` and `following` that when i delete a subcription/following, it deletes the associated user as well. Maybe the cascade is going in the opposite direction from what we wanted?

Reply:
Yes it is going in the opposite direction - we should only have `cascade='all,delete'` for relationships that define related model that should be deleted if the base model is deleted.

Reply:
I think from working quickly we might have a couple of these issues - we should have a PR to fix those.  Merge this and I'll make that PR.